src/content/warnings/react-test-renderer.md

```diff
@@ -0,0 +1,7 @@
+---
+title: react-test-renderer Deprecation Warning
+---
+
+react-test-renderer is deprecated. A warning will fire whenever calling ReactTestRenderer.create() or ReactShallowRender.render(). The react-test-renderer package will remain available on NPM but will not be maintained and may break with new React features or changes to React's internals.
+
+The React Team recommends migrating your tests to [@testing-library/react](https://testing-library.com/docs/react-testing-library/intro/) for a modern and well supported testing experience.
```

Comment:
I'll add the link to clarify. Overall we're going to come back to this messaging soon once we've agreed upon the plan going forward. And we may have separate warnings in RN for introspection API usage rather than during create/render.